article/_posts/2019-05-30-data-fusion-remote-sensing.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: review
+title:  "Multisource and Multitemporal Data Fusion in Remote Sensing"
+tags:   deep-learning segmentation remote-sensing CNN hyperspectral multi-task-learning point-cloud
+author: Charles Authier
+pdf:  https://arxiv.org/pdf/1812.08287.pdf
+cite:
+    authors: "Pedram Ghamisi, Behnood Rasti, Naoto Yokoya, Qunming Wang, Bernhard Hofle, Lorenzo Bruzzone, Francesca Bovolo, Mingmin Chi, Katharina Anders, Richard Gloaguen, Peter M. Atkinson, Jon Atli Benediktsson"
+    title:   "Multisource and Multitemporal Data Fusion in Remote Sensing"
+    venue:   "arXiv preprint arXiv:1812.08287. 2018 Dec 19."
+---
+
+*KEYS WORDS: light detection and ranging (LiDAR), terrestrial laser scanning (TLS), synthetic aperture radar (SAR), Gravity Recovery And Climate Experiment (GRACE), Medium Resolution Imaging Spectrometer (MERIS), electromagnetic spectrum (EMS) *
+
+
+This paper brings together the advances of multisource and multitemporal data fusion approaches with different research communities and provides a thorough and discipline-specific starting point for researchers at different levels.
+More specifically, it provides an overview of many important contributions dedicated to the topics of pansharpening and resolution enhancement, point cloud data fusion, hyperspectral and LiDAR data fusion, multitemporal data fusion, as well as big data and social media.
+
+The remote sensors onboard the multiples platforms may vary greatly in multiple dimensions.
+For example, the types of properties sensed and the spatial and spectral resolutions of the data.
```

Comment:
Does not seem like the sentence is finished. Does it refer to the picture, to the previous sentence? If it is the latter, may be 

*The remote sensors onboard multiple platforms may vary greatly in multiple dimensions (e.g. types of properties sensed, spatial and spectral resolutions of the data, etc.).*


is clearer?

Reply:
Done in 8b8f8b3.